test/func_network.py

```diff
@@ -85,9 +85,16 @@ def network_pktdriver_mtcp(self, driver):
 $_tapdev = "tap0"
 """, timeout=30)
 
-    p.join()
-    sleep(5)
-    p.close()
+    p.join(timeout=45)
+    if p.is_alive():
+        stderr.write("Timeout on join() for little_webserver - killing ")
+        stderr.flush()
+        p.kill()
+        p.join(timeout=5)
```

Comment:
Maybe you should assert if
this one timed-out? As I don't
think it should.

Reply:
I'm not sure any process can resist a `kill -9` but just in case I've now raised an exception should that occur. In addition I've revived the close() just to be explicit.